test/test_find.cpp

```diff
@@ -19,8 +19,9 @@
 #define ENABLE_LOGGING 1
 
 #if ENABLE_LOGGING
-#define LOG(expected, actual) { \
-    printf("Expected: %zu Actual: %zu\n", expected, actual);}
+#define LOG(expected, actual) do { \
+    printf("Expected: %zu Actual: %zu\n", expected, actual); \
+  } while (0)
 #else
 #define LOG(X, arg) {}
```

Comment:
@yenWu the same reasoning would apply here when `ENABLE_LOGGING` is defined as 0, right?

Reply:
@dhood ,Yes. And thank @clalancette for fixing it up.